### PR TITLE
Make impermanence a flake

### PR DESCRIPTION
**Pull request by danieldk**
Thanks for this very nice module! This small change makes this repository usable as a flake.

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  inputs.nixpkgs.url = "nixpkgs/nixos-20.03-small";
+
+  outputs = { self, nixpkgs }: {
+    nixosModule = self.nixosModules.impermanence;
+    nixosModules.impermanence = import ./nixos.nix;
```
Comment (talyz) [inline flake.nix:6]: Are both necessary? The wiki entry makes it sound like `nixosModules` is only needed if multiple modules are provided.

**Review by talyz — COMMENTED** [review]
Thanks a lot for this! I have yet to try out flakes, so I obviously haven't tried this, but it looks good to me: I only have one question.

**Comment by bbigras** [comment]
Any progress on this?

**Comment by cole-h** [comment]
I took the liberty of resolving the above question by using `nixosModules.impermanence` over `nixosModule` (this is what e.g. `home-manager` and `Nixpkgs` use).

I also updated the `nixpkgs` input to point to 20.09, since that is the new stable. PTAL.

**Comment by bbigras** [comment]
Thanks everyone!

**Pull request merged**